packages/build/tests/plugins/outputs/fixtures/basic/netlify.yml

```diff
@@ -0,0 +1,5 @@
+plugins:
+  - package: ./two
+    config:
+      test: ${{outputs:one.example}}
```

Comment:
One note: Because the same plugin can be added multiple times with different config, I think this might need a logical ID.

```
  - package: ./one
     id: one
```

Example

```
plugins:
  - package: ./two
    config:
      test: ${{outputs:one.example}}
      other: ${{outputs:three.example}}
  - package: ./one
     id: one
  - package: ./one
     id: three
```

Reply:
Yes, this is how this is currently working.
Users can specify an `id` property, and the `${outputs:...}` uses that for targeting.
The property defaults to the plugin `name` property, which is why it is omitted in that example.

Please note the double `${{...}}` syntax is a temporary hack due to some regression issue I encountered, it's not the final syntax.